db-init/src/main/resources/flyway.conf

```diff
@@ -6,8 +6,8 @@ flyway.connectRetries=10
 flyway.defaultSchema=${FLYWAY_SCHEMA}
 flyway.schemas=${FLYWAY_SCHEMA}
 flyway.createSchemas=true
-flyway.table=${FLYWAY_TABLE}
-flyway.tablespace=${FLYWAY_TABLESPACE}
+flyway.table=schema_history
+flyway.tablespace=pg_default
```

Comment:
This is the only place these configuration setting are needed, and the setting values do not need to be shared with other containers.